SemiLeptonicWVA/Utils/src/GenEventInfo.cc

```diff
@@ -264,6 +270,23 @@ GenEventInfo::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 	  PUWeight = LumiWeights_.weight( Tnpv );
 	}
 
+
+
+	edm::Handle< LHEEventProduct > LHEEventInfo;
+        iEvent.getByLabel("externalLHEProducer", LHEEventInfo);
```

Comment:
Remember that in the MiniAOD, the data is under the label "source" instead of "externalLHEProducer" (check email from Rafael regarding this).  Change to iEvent.getByLabel("source", LHEEventInfo);